compiler/src/iree/compiler/Tools/BUILD.bazel

```diff
@@ -178,20 +178,14 @@ iree_compiler_cc_library(
         "IREE_HAVE_ROCM_TARGET",
         "IREE_HAVE_VMVX_TARGET",
         "IREE_HAVE_VULKANSPIRV_TARGET",
-    ] + select({
-        ":cuda_enabled": ["IREE_HAVE_CUDA_TARGET"],
-        "//conditions:default": [],
-    }),
+    ],
```

Comment:
It looks like this breaks Cuda in Bazel again? The Bazel macro `iree_compiler_register_plugin` is a noop: https://github.com/openxla/iree/blob/afff73aa4004ee8a9ba6a6fc434a1ba4deba3705/build_tools/bazel/build_defs.oss.bzl#L76

Reply:
Yeah, that is one of the things I need to fix before this pr is done. Missed it before firing off.